src/pkg/clients/S3_dummy.go

```diff
@@ -28,3 +29,7 @@ func (s s3ClientDummy) GetObject(ctx context.Context, id string) (io.Reader, err
 func (s s3ClientDummy) PutObjectInput(ctx context.Context, f io.Reader, title string) error {
 	return s.putObjectInput(f, title)
 }
+
+func (s s3ClientDummy) CreateBucketIfDoesNotExists(ctx context.Context, bucketName string) error {
+	return s.createBucket(bucketName)
```

Comment:
The name createBucket should not be createBucketIfDoesNotExists ? for the other functions we used this standard